### PR TITLE
#5665 Script Readme

### PR DESCRIPTION
**Pull request by ericsorenson**
First Draft

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -70,13 +70,13 @@ Once [you have Docker installed](https://docs.docker.com/install/), the followin
...
 - You can access the public UI at http://localhost:5678
-- You can access the API at http://localhost:3000
+- You can access the API at http://localhost:4000
 - You can access the DynamoDB shell at http://localhost:8000/shell
 - You can access the DynamoDB admin UI at http://localhost:8001
 - You can access S3 local at http://localhost:9000
 - You can access the style guide at http://localhost:1234/style-guide
 
-Within Docker, you should allocate 4 CPUs, 16 GB of RAM, and 4 GB of swap. With fewer resources, the software is likely to fail to run with errors that don’t make it obvious what the problem is.
+Within Docker, you should allocate 2+ CPUs, 8+ GB of RAM, and 4+ GB of swap. With fewer resources, the software is likely to fail to run with errors that don’t make it obvious what the problem is.
```
Comment (adunkman) [inline README.md:79]: :tada:

File: docs/SCRIPTS.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Deploys the UI code in your workspace to AWS.
+
+`npm run reset-cases ENV`
+
+Removes all cases from DynamoDB and ElasticSearch for an environment in AWS.
+
+`npm run destroy:ENV`
+
+Removes all infrastructure resources and data for an environment in AWS.
```
Comment (adunkman) [inline docs/SCRIPTS.md:39]: I get a little cross-eyed trying to see if the description applies to the script below or above the line — maybe a table or bulleted list would be more clear?

File: web-client/src/views/CaseDetail/AddressDisplay.jsx [hunk 3]
```diff
@@ -35,6 +35,7 @@ export const AddressDisplay = connect(
             <span>
               <br />
               c/o {contact.inCareOf}
+              {contact.title && <span>, {contact.title}</span>}
```
Comment (adunkman) [inline web-client/src/views/CaseDetail/AddressDisplay.jsx:38]: I think this change might have snuck into this PR… 🐌

**Review by adunkman — COMMENTED** [review]
Excited dancing hotdog wearing DJ headphones

I didn’t know about some of these scripts — thanks so much for the documentation! 💯 

Dropped a few inline comments below.

File: docs/SCRIPTS.md (hunk 2 above)
Comment (ericsorenson) [reply docs/SCRIPTS.md:39]: Updated!

**Pull request merged**